commit 37637a37ac8b606633cd574649bfe7a034f5868a (refs/changes/72/372/2)
Author: Joshua M. Clulow <jmc@joyent.com>
Date:   2016-08-29T20:59:44+00:00 (3 years, 1 month ago)
    
    TOOLS-1535 gerrit could be better with names and faces
    Reviewed by: Alex Wilson <alex.wilson@joyent.com>
    Reviewed by: Dave Pacheco <dap@joyent.com>
    Approved by: Dave Pacheco <dap@joyent.com>

diff --git a/images/appserver/etc/gerrit.config.base b/images/appserver/etc/gerrit.config.base
index d1e6159..ef22120 100644
--- a/images/appserver/etc/gerrit.config.base
+++ b/images/appserver/etc/gerrit.config.base
@@ -37,6 +37,18 @@
 [cache]
 	directory = cache
 
+#
+# The "web_sessions" cache tracks open user sessions.  These effectively map a
+# cookie in your browser back to an identity.  The maximum age of entries in
+# this cache determines how often a user must sign in again.
+#
+# Note that the Gerrit documentation suggests that a value of 0 will allow
+# entries to remain in the cache until it fills.  This appears to not to be the
+# case in practice.  Instead, we have selected a large positive value.
+#
+[cache "web_sessions"]
+	maxAge = 1 y
+
 [plugins]
 	allowRemoteAdmin = true
 
